common-rest/src/main/java/org/uniprot/api/common/repository/store/StreamerConfigProperties.java

```diff
@@ -14,6 +14,7 @@
     private int storeBatchSize;
     private int storeFetchMaxRetries;
     private int storeFetchRetryDelayMillis;
+    private int storeMaxCountToRetrieve;
 
     private String zkHost;
     private String idFieldName;
```

Comment:
Hi Edd do you think we could reuse Solr collection that we have in this class in the TupleStreamTemplate validation method? I pointed out bellow in TupleStreamTemplate